ui/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Do we leave the MIT License or replace it with the standard Google Apache License?

Reply:
FYI MIT license allows re licensing, so possibly we can just move forward and license the whole project under Apache?

Reply:
project is already licensed under apache, should use apache header. I think the exact source header snippit is in the readme

Reply:
yeah, it was just that the scaffold OpenWC provides included the license causing this discussion. removed and added headers in a future commit.